examples/idv/src/controllers/index.controller.with.face.comparison.js

```diff
@@ -0,0 +1,117 @@
+const {
+  IDVClient,
+  SessionSpecificationBuilder,
+  RequestedLivenessCheckBuilder,
+  RequestedFaceMatchCheckBuilder,
+  SdkConfigBuilder,
+  RequestedFaceComparisonCheckBuilder,
+  CreateFaceCaptureResourcePayloadBuilder,
+  UploadFaceCaptureImagePayloadBuilder,
+} = require('yoti');
+const config = require('../../config');
+
+/**
+ * Create an IDV session.
+ */
+async function createSession() {
+  const idvClient = new IDVClient(config.YOTI_CLIENT_SDK_ID, config.YOTI_PEM);
+
+  const sessionSpec = new SessionSpecificationBuilder()
+    .withClientSessionTokenTtl(600)
+    .withResourcesTtl(90000)
+    .withUserTrackingId('some-user-tracking-id')
+
+  // For zoom liveness check [only one type of liveness check to be enabled at a time]
+    .withRequestedCheck(new RequestedLivenessCheckBuilder()
+      .forZoomLiveness()
+      .build())
+  // For static liveness check
+  // .withRequestedCheck(
+  //   new RequestedLivenessCheckBuilder()
+  //     .forStaticLiveness()
+  //     .build()
+  // )
+    .withRequestedCheck(new RequestedFaceMatchCheckBuilder()
+      .withManualCheckNever()
+      .build())
+
+    .withRequestedCheck(new RequestedFaceComparisonCheckBuilder()
+      .withManualCheckNever()
+      .build())
+
+    .withSdkConfig(new SdkConfigBuilder()
+      .withAllowsCameraAndUpload()
+      .withPrimaryColour('#2d9fff')
+      .withSecondaryColour('#FFFFFF')
+      .withFontColour('#FFFFFF')
+      .withLocale('en-GB')
+      .withPresetIssuingCountry('GBR')
+      .withSuccessUrl(`${config.YOTI_APP_BASE_URL}/success`)
+      .withErrorUrl(`${config.YOTI_APP_BASE_URL}/error`)
+      .withPrivacyPolicyUrl(`${config.YOTI_APP_BASE_URL}/privacy-policy`)
+      .withAllowHandoff(true)
+      .withEarlyBiometricConsentFlow() // or withJustInTimeBiometricConsentFlow()
+      .withIdDocumentTextExtractionGenericRetries(5)
+      .withIdDocumentTextExtractionReclassificationRetries(5)
+      .build())
+
+    .build();
+
+  return idvClient.createSession(sessionSpec);
+}
+
+async function addFaceCaptureResourceToSession(sessionId) {
+  const idvClient = new IDVClient(config.YOTI_CLIENT_SDK_ID, config.YOTI_PEM);
+
+  const sessionConfiguration = await idvClient.getSessionConfiguration(sessionId);
+
+  console.log('>>>>', sessionConfiguration, '<<<< sessionConfiguration');
```

Comment:
Do we want to keep all these logs around?

Reply:
I think these were adding for testing only and can be removed, right @laurent-yoti ?

Reply:
yeah, just temporary, let's remove them